app/src/main/java/com/iic/shopingo/ui/trip_flow/activities/ShoppingListActivity.java

```diff
@@ -0,0 +1,104 @@
+package com.iic.shopingo.ui.trip_flow.activities;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.ListView;
+import com.iic.shopingo.ui.trip_flow.data.ShoppingList;
+import com.iic.shopingo.ui.trip_flow.views.ShoppingListItem;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by asafg on 04/03/15.
+ */
+public class ShoppingListActivity extends Activity implements ShoppingListItem.OnCallListener {
+  private ListView list;
+  private ShoppingListAdapter adapter;
+
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    list = new ListView(this);
+    setContentView(list);
```

Comment:
Essentially, yes.
There's also ListActivity that you can inherit from that provides a built-in listview, but it looks more annoying than it's worth.